LR_Tourist/TouristWebApp/wwwroot/lib/bootstrap/LICENSE

```diff
@@ -0,0 +1,22 @@
+The MIT License (MIT)
```

Comment:
The `lib` folder should be removed from the repository.

Reply:
fixed